weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/OverflowMenu.tsx

```diff
@@ -101,11 +108,21 @@ DialogActions.displayName = 'S.DialogActions';
 
 const MAX_DELETED_CALLS_TO_SHOW = 10;
 
-const ConfirmDeleteModal: FC<{
-  calls: CallSchema[];
+export const ConfirmDeleteModal: FC<{
+  entity: string;
+  project: string;
+  calls: Array<{callId: string; name: string}>;
```

Comment:
seems odd to create a new shape for call - why not just pass in the CallSchema and parse out the name inside? Feels odd for the caller to do this work

Reply:
This allows for a bit more flexibility, by only requiring exactly what we need, but I do also see your point. Especially with the comment below. This is of course how it works right now, so I will just not include this change in the pr...